xbmc/cores/VideoPlayer/DVDInputStreams/InputStreamAddon.cpp

```diff
@@ -31,7 +31,6 @@ CInputStreamAddon::CInputStreamAddon(const CFileItem& fileitem, std::shared_ptr<
 CInputStreamAddon::~CInputStreamAddon()
 {
   Close();
-  m_addon->Stop();
```

Comment:
No becomes not used on Inputstream addon's. By my look over the addons is the Stop only used on Screensaver and Visualization.

Currently I go over all addons to check. The pull requests to them comes this time after them is in, are to much to make before.